contracts/deploy-ready/ERC20DeployReady.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.6.0;
+
+import "../access/AccessControl.sol";
+import "../GSN/Context.sol";
+import "../token/ERC20/ERC20.sol";
+import "../token/ERC20/ERC20Burnable.sol";
+import "../token/ERC20/ERC20Pausable.sol";
+
+/**
+ * @dev Full-fledged {ERC20} token, including:
+ *
+ *  - ability for holders to burn (destroy) their tokens
+ *  - a minter role that allows for token minting (creation)
+ *  - a pauser role that allows to stop all token transfers
+ *
+ * The account that deploys the contract will be granted the minter role, the
+ * pauser role, and the default admin role, meaning it will be able to grant
+ * both the minter and pauser roles. See {AccessControl} for details.
+ *
+ * This contract serves as an example of how to integrate different {ERC20}
+ * modules and use the {AccessControl} contract to create different permissions.
+ * It can be deployed as-is for quick prototyping or testing.
```

Comment:
It can also be used in production... We shouldn't frame these as simply examples. Not using these should be for intermediate-advanced usage.